### PR TITLE
chore: standardize npm script names

### PR DESCRIPTION
**Pull request by snitin315**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

#### What changes did you make? (Give an overview)

Refers #14827

This updates the names of the scripts in package.json to be consistent with the [new standard](https://eslint.org/docs/latest/contribute/package-json-conventions).

#### Is there anything you'd like reviewers to focus on?
Is there any script we need to update in Netlify/jenkins?

### CODE REVIEW

**Review by amareshsm — APPROVED** [review]
LGTM. Waiting for code owner's review.

File: docs/package.json
Comment (nzakas) [inline docs/package.json:22]: Maybe this should also be `build`?

```suggestion
        "build:images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
```

**Review by nzakas — CHANGES_REQUESTED** [review]

File: docs/package.json
Comment (harish-sethuraman) [reply docs/package.json:22]: I feel like we could have it as `minify:images` only? or it could be `build:images:minify`?

Comment (snitin315) [reply docs/package.json:22]: We can update it to `build:minify-images` ,WDYT?

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks! Leaving open for others to review.

**Comment by mdjermanovic** [comment]
> Is there any script we need to update in Netlify/jenkins?

I think we don't need to update anything on Netlify/Jenkins. Netlify only runs `npm run build`. I believe neither of the scripts from this package,json is run during the release process on Jenkins.

**Review by harish-sethuraman — APPROVED** [review]
LGTM. leaving it open for @nzakas's review

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**
